apps/backend/src/mutations/UserMutations.ts

```diff
@@ -99,7 +99,7 @@ export default class UserMutations {
     }
     // Check if email exist in database and if user has been invited before
     const user = await this.dataSource.getByEmail(args.email);
-    if (user && user.placeholder) {
+    if (user) {

```

Comment:
issue: I think this is fine, only now the`else if (user)`clause is will never be called, we should probably clean it up